File: content/1-basics/3-conditionals/hs/conditionals.hs
main :: IO ()
main = do
    putStrLn "Enter a number:"
    input <- getLine
    let number = read input :: Int
    if number > 0
        then putStrLn "The number is positive."
        else if number < 0
            then putStrLn "The number is negative."
            else putStrLn "The number is zero."

File: content/1-basics/3-conditionals/rs/conditionals.rs
use std::io;

fn main() {
    println!("Enter a number:");

    let mut input = String::new();
    io::stdin().read_line(&mut input).expect("Failed to read line");

    let number: i32 = input.trim().parse().expect("Please enter a valid number");

    if number > 0 {
        println!("The number is positive.");
    } else if number < 0 {
        println!("The number is negative.");
    } else {
        println!("The number is zero.");
    }
}